src/components/UserSettingsModal/UserSettingsModal.tsx

```diff
@@ -23,23 +31,23 @@ export class UserSettingsModal extends React.Component<
     super(props);
     this.state = {};
   }
-  componentWillMount() {
-    const userDataPromise = getUserData(1);
-
-    userDataPromise
-      .then(response => {
-        console.log(response);
-      })
-      .catch(err => console.log(err));
-  }
 
   render() {
     return (
-      <div className="user-settings-modal">
+      <div
+        className={
+          'user-settings-modal ' +
+          (this.props.isOpen ? 'user-settings-modal--is-open' : '')
+        }
+      >
+        <div
+          className="user-settings-modal__background"
+          onClick={this.props.onClick}
+        />
         <div className="user-settings-modal__box">
           <figure className="user-settings-modal__avatar">
             <img
-              src="https://pbs.twimg.com/profile_images/1013798240683266048/zRim1x6M_400x400.jpg"
+              src={host + this.props.stores.assets.avatar.filePath}
```

Comment:
Add helper for get the url.

Reply:
It's for now. I will return from backend hostName with path as one;